contentcuration/contentcuration/frontend/accounts/pages/ActivationSent.vue

```diff
@@ -0,0 +1,42 @@
+<template>
```

Comment:
Organization-wise, we've been putting components under a `views` folder. To prevent potential loss of history if we decide to standardize on this later, could you rename this folder to views for consistency?

Reply:
Maybe it wasn't clear, but the proposal I made during the meeting was to replace 'views' with two folders:

* pages
* components

If you don't like this convention we can reconsider. Bit in any case, there is no loss of history in git if files move?

Reply:
> But in any case, there is no loss of history in git if files move?

It will only detect a move vs. a delete if the file remains completely the same (even with `git mv`), but I should be able to get around that (last time, things like linting errors and references not being found made it so I had to change the files, but this probably won't be the case here). 

> replace 'views' with two folders

How do you see sub-pages like move being organized in this structure? Something like this?
```
channelEdit
- components
- pages
  | - move
      | - pages
      | - components
```
Or would move be at the same level as channelEdit?

Reply:
Decided that we will put page-specific components under the pages/[page] folder